custom_components/smartthinq_sensors/sensor.py

```diff
@@ -329,37 +329,37 @@
         ATTR_MEASUREMENT_NAME: "Cooktop Left Front",
         ATTR_ICON: DEFAULT_ICON,
         ATTR_VALUE_FEAT: FEAT_COOKTOP_LEFT_FRONT_STATE,
-        ATTR_ENABLED: True,
+        ATTR_ENABLED: False,
```

Comment:
Just remove the attribute, False is default value.

Reply:
Done.